### PR TITLE
Code scanning

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

File: spec/models/feed_spec.rb [hunk 1]
```diff
@@ -29,10 +29,6 @@
   end
 
   it "is not expired when new" do
-    expect(feed.created_at).to be_nil
-    expect(feed.fetched_at).to be_nil
-    expect(feed.expired?).to be_falsey
```
Comment (numist) [inline spec/models/feed_spec.rb:34]: is `created_at` set in `new` now? I had this all in because I crashed myself by checking `expired?` before `save`

**Review by numist — COMMENTED** [review]
Looks good, just one quick question about removed test code.

File: spec/models/feed_spec.rb (hunk 1 above)
Comment (indirect) [reply spec/models/feed_spec.rb:34]: The timestamp fields (created_at/updated_at) are only set the first time the object is saved. Line 35 calls `expired?` before we save, so this still checks that.

Comment (numist) [reply spec/models/feed_spec.rb:34]: but now we don't have a test for when `fetched_at` is nil and `created_at` is set?

Comment (indirect) [reply spec/models/feed_spec.rb:34]: ugh apparently I did this twice and this was the bad version. oops.

Comment (indirect) [reply spec/models/feed_spec.rb]: took it out!

**Review by numist — APPROVED** [review]

**Pull request merged**